doc/keymap-format-text-v1.md

```diff
@@ -535,7 +535,9 @@ Comments are introduced following either `//` or `#` until the end of the line.
 
   | Escape sequence    | Meaning                                                  |
   | ------------------ | -------------------------------------------------------- |
+  | `\&`               | Produce no character (`"\&"` denotes an empty string) but enable to stop a *previous* escape sequence, e.g. `\61\&2` produces “12” whereas `\622` is invalid. |
```

Comment:
Do we have a use case for this? In case someone really has this problem, it *is* possible to do `\00127`  IIUC.

Reply:
It made sense for the initial `\uNNNN` escape proposal, but the current proposal use `\u{NNNN}` so `\&` is only useful for octal sequences, a niche feature. I still prefer `\42\&2` over `\00422`.

I removed this feature.